### ISSUE TITLE
[CI] InternalStatsTests.testFromXContent Failure

### ISSUE THREAD

**Issue by davidkyle** [elastic/elasticsearch#27245]
And `InternalStatsBucketTests.testFromXContent`

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+multijob-windows-compatibility/865/console 

This reproduces:
```
REPRODUCE WITH: gradle :core:test \
  -Dtests.seed=5858C05C5DF7BC20 \
  -Dtests.class=org.elasticsearch.search.aggregations.metrics.InternalStatsTests \
  -Dtests.method="testFromXContent" \
  -Dtests.security.manager=true \
  -Dtests.locale=hu \
  -Dtests.timezone=Canada/Mountain

REPRODUCE WITH: gradle :core:test \
  -Dtests.seed=5858C05C5DF7BC20 \
  -Dtests.class=org.elasticsearch.search.aggregations.metrics.InternalStatsBucketTests \
  -Dtests.method="testFromXContent" \
  -Dtests.security.manager=true \
  -Dtests.locale=zh-CN \
  -Dtests.timezone=Europe/Uzhgorod
```

Error output is 
```
FAILURE 0.68s | InternalStatsBucketTests.testFromXContent <<< FAILURES!
   > Throwable #1: java.lang.AssertionError: Error when comparing xContent.
   >             stats_bucket#petIa: 
   >                              avg: expected [null] but not found
   >                            count: same [0]
   >                              max: expected [null] but not found
   >                              min: expected [null] but not found
   >                              sum: expected [0.0] but not found
```

Possibly related to #27193

**Comment by cbuescher** [comment]
In  #27193 the xContent output of InternalStats was changes, this change needs to be reflected in the corresponding ParsedStats as well. Will push a fix shortly.

**Comment by cbuescher** [comment]
Fixed by 9abc26ee92a504dbaec9aa89469603f2113e5432
